<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderBookDialog</class>
 <widget class="QDialog" name="OrderBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sHiFT Orderbook</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="StockListLayout">
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="StockListLabel">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>Stock List:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="SearchLayout">
         <item>
          <widget class="QLineEdit" name="SearchSymbol">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Search Symbol</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="StockList">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="LastPriceGrid">
         <property name="rightMargin">
          <number>100</number>
         </property>
         <item row="1" column="1">
          <widget class="QLabel" name="USD">
           <property name="text">
            <string>USD</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="DiffPercentLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="AcknowlageLabel">
           <property name="font">
            <font>
             <pointsize>6</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Relative to open price. Market open.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="PriceDifferenceLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="LastPriceLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="ArrowLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="TickerNameLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="GlobalBox">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Global</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="GlobalOrderbook">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="GlobalBid">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Bid</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QTableView" name="GlobalBidTable">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>100</number>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="GlobalAsk">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Ask</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QTableView" name="GlobalAskTable">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="LocalBox">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Local</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="LocalOrderbook">
            <item>
             <widget class="QGroupBox" name="LocalBid">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Bid</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QTableView" name="LocalBidTable">
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="LocalAsk">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Ask</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QTableView" name="LocalAskTable">
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
